front/components/assistant/AssistantDetailsDropdownMenu.tsx

```diff
@@ -25,40 +26,39 @@ import { useAgentConfiguration } from "@app/lib/swr/assistants";
 import { useUser } from "@app/lib/swr/user";
 
 interface AssistantDetailsDropdownMenuProps {
-  agentConfigurationId: string;
+  agentConfiguration: LightAgentConfigurationType;
   owner: WorkspaceType;
   variant?: "button" | "plain";
-  onAgentDeletion?: () => void;
+  canDelete?: boolean;
+  showAssistantDetails?: () => void;
   showAddRemoveToList?: boolean;
 }
 
 export function AssistantDetailsDropdownMenu({
-  // The `agentConfiguration` cannot be used directly as it isn't dynamically
-  // updated upon user list mutations. This limitation stems from its
-  // propagation method from <ConversationMessage>.
-  agentConfigurationId,
+  agentConfiguration,
```

Comment:
the previous situation was relevant when we were able to add / remove from user-list.